pkg/operator/ceph/csi/util.go

```diff
@@ -99,3 +102,24 @@ func templateToDeployment(name, templatePath string, p templateParam) (*apps.Dep
 	}
 	return &ds, nil
 }
+
+func getPortFromENV(env string, defaultPort uint16) uint16 {
```

Comment:
What about decoupling the conditions to make this clearer?

if port == "" --> return default
parse port --> if error --> return default
if p > 65535 --> return default

Then the end of the function can return the `port` correctly.